includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -880,8 +889,16 @@ public function ajax_update_shipping_method() {
 
 		WC()->cart->calculate_totals();
 
+		$should_show_itemized_view = function() {
```

Comment:
I'm curious - why not do this imperatively?

Reply:
Fixed in the latest rebase https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1239/commits/a9fca6808cc3d34218b5d4975b56a876e6b8bda0 🙂 

This is a C++ habit bleeding into my PHP development here 😅 . See [this talk](https://github.com/CppCon/CppCon2017/tree/master/Presentations/10%20Core%20Guidelines%20You%20Need%20to%20Start%20Using%20Now) from cppcon 2017.